source/_page_templates/ability_template.rst

```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
```

Comment:
Element/subelement?

Reply:
I still think this should be explicitly stated even though the ability will be under an elemental category.